engine-rest/engine-rest/src/main/java/org/camunda/bpm/engine/rest/impl/TelemetryRestServiceImpl.java

```diff
@@ -43,4 +46,12 @@ public TelemetryConfigurationDto getTelemetryConfiguration() {
     Boolean telemetryEnabled = managementService.isTelemetryEnabled();
     return new TelemetryConfigurationDto(telemetryEnabled);
   }
+
+  @Override
+  public String getTelemetryData() {
+    ManagementService managementService = processEngine.getManagementService();
+    TelemetryData data = managementService.getTelemetryData();
+
+    return new Gson().toJson(data);
```

Comment:
This would mean we need to duplicate the serialization instructions on the DTOs ([example](https://github.com/camunda/camunda-bpm-platform/blob/b1602f804cc000c0b4d7cd552f33ff59e3db226d/engine/src/main/java/org/camunda/bpm/engine/impl/telemetry/dto/InternalsImpl.java#L38)). Should we then think about using Jackson there as well? Do you know why we opted for Gson?

Reply:
In the engine we have Gson for JSON serialization, not Jackson. We started using it when we refactored the serialization of task filter queries from json.org to a different library. Gson is shaded into the engine.